Core/Src/main.c

```diff
@@ -690,6 +696,18 @@ static void MX_GPIO_Init(void)
 
 /* USER CODE BEGIN 4 */
 
+void watchdog_init(void)
+{
+   HAL_GPIO_WritePin(Watchdog_Out_GPIO_Port, Watchdog_Out_Pin, GPIO_PIN_SET);
+}
+void watchdog_pet(void)
+{
+   //second iteration maybe redundant, datasheet unclear so we pet twice
+    HAL_GPIO_WritePin(Watchdog_Out_GPIO_Port, Watchdog_Out_Pin, GPIO_PIN_RESET);
```

Comment:
I'm gonna trust you on writing to it twice lol. My only concern is that there isn't a delay so it might be too fast for the watchdog to see, but I think its an issue that we will have to address when putting it on hardware